demisto_sdk/commands/generate_outputs/generate_descriptions/generate_descriptions.py

```diff
@@ -0,0 +1,289 @@
+
+
+def write_desc(c_index, final_output, o_index, output_path, verbose, yml_data):
+    if verbose:
+        print(f"Writing: {final_output}\n---")
+    yml_data['script']['commands'][c_index]['outputs'][o_index][
+        'description'] = final_output
+    write_yml(output_path, yml_data)
+
+
+def correct_interactively(command_ctx, final_output, output):
+    print('\r', end='', flush=True)
+    final_output = pinput('>> Generated output: ',
+                          final_output)
+    # Learn from mistakes
+    if final_output != output:
+        y = input("Should we append to prompt (y/n)? ")
+        if y == 'y':
+            print(
+                f"(Correcting prompt `{final_output}` instead of `{output}`)")
+            correct_prompt(command_ctx, final_output)
+    return final_output
+
+
+def generate_ai_descriptions(
+        input_path: str,
+        output_path: str = "out.yml",
+        interactive: bool = True,
+        verbose: bool = False,
+        insecure: bool = False,
+):
+    """ Generate integration command contexts.
+
+    Args:
+        input_path: path to the yaml integration input path
+        output_path: path to the yaml integration output path
+        interactive: interactivity (correct ai result mistakes)
+        verbose: verbose (debug mode)
+        insecure: insecure https (debug mode)
+    """
+    print()
+    print("⚠" * 100)
+    print('''███████ ██   ██ ██████  ███████ ██████  ██ ███    ███ ███████ ███    ██ ████████  █████  ██
+██       ██ ██  ██   ██ ██      ██   ██ ██ ████  ████ ██      ████   ██    ██    ██   ██ ██
+█████     ███   ██████  █████   ██████  ██ ██ ████ ██ █████   ██ ██  ██    ██    ███████ ██
+██       ██ ██  ██      ██      ██   ██ ██ ██  ██  ██ ██      ██  ██ ██    ██    ██   ██ ██
+███████ ██   ██ ██      ███████ ██   ██ ██ ██      ██ ███████ ██   ████    ██    ██   ██ ███████
+''')
+    print("this feature is experimental, use with caution.")
+    print("⚠" * 100)
+    print()
+
+    try:
+        similar_paths: Dict[str, str] = {}
+        yml_data = get_yaml(input_path)
+
+        # Fix sometimes the yml doesn't contain the full structure
+        #  (json-to-outputs)
+        if 'script' not in yml_data:
+            yml_data = {'script': {'commands': [yml_data]}}
+
+        commands = yml_data.get("script", {})
+        commands = commands.get('commands', [])
+
+        # Iterate over every command
+        for c_index, command in enumerate(commands):
+            command_name = command.get('name')
+
+            if interactive or verbose:
+                print(f'Command: {command_name}')
+
+            outputs = command.get('outputs')
+            if not outputs:
+                if interactive or verbose:
+                    print("-- Skipping because no outputs for command")
+                continue
+
+            # Iterate over every output per command
+            for o_index, o in enumerate(outputs):
+                command_ctx = o.get('contextPath')
```

Comment:
name of command_ctx is a little confusing IMO. Because it isn't the context of the command, more like of a command_output_path
```suggestion
                command_output_path = o.get('contextPath')
```